openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -191,6 +192,31 @@ def test_serializer_save_fails(self, serializer_save):
         pending_change = PendingEmailChange.objects.filter(user=self.user)
         self.assertEqual(0, len(pending_change))
 
+    def test_language_proficiency_eventing(self):
+        """
+        Test that eventing of language proficiencies, which happens update_account_settings method, behaves correctly.
+        """
+        def verify_event_emitted(new_value, old_value):
+            update_account_settings(self.user, {"language_proficiencies": new_value})
+            self.assert_event_emitted(
+                USER_SETTINGS_CHANGED_EVENT_NAME, setting="language_proficiencies",
+                old=old_value, new=new_value, user_id=self.user.id, table="student_languageproficiency"
+            )
+            self.reset_tracker()
+
+        # First, test that no event is emitted if language_proficiencies is not included.
+        update_account_settings(self.user, {"year_of_birth": 900})
+        account_settings = get_account_settings(self.user)
+        self.assertEqual(900, account_settings["year_of_birth"])
+        self.assert_no_events_were_emitted()
```

Comment:
This is failing now because changing the year_of_birth will emit an event in util.model_utils.tracker (previously, language_proficiencies was using its own tracker).

I think we can just remove lines 207-211. If extra events were being fired, we have plenty of tests that would catch them.